### DISCUSSION TITLE
Can we make Flips more semi-unique?

### DISCUSSION THREAD

**Discussion by ubiubi18** [idena-network/idena-go#962] [Ideas]
I think there is an aspect of idena cryptonomics that seems to incentify centralised behaviour that harms the network security. 

Human farms do require farm workers only for a small fraction of 10 to 15 minutes to solve flips,  that accounts for only 20 % (validation rewards) to 35 % (optional validation rewards + report rewards) of possible session rewards. But no matter that they dont need them at all to produce flips.

Those 10 minutes solving flips are pretty cheap in production costs if you have for example an internetcafe full of dudes solving it for free access to PCs for a few hours. It would be much more expensive to ask those people to wait idle for another 30 to 60 minutes until session is over to let them create new flips. 

On the other hand creating valid flips accounts for another 30% of rewards! 
So the logical consequence is that most farm owners are going to extract that flip rewards themself, or let it one or few specialized workers do. They are at no point forced to involve and pay all farm workers for more than 10 minutes.

If we look at the blockexplorer how and when human farms did submit their flips in the past, it seems maybe a few of them use preproduced flips with just one keyword and submit it, hoping not to be reported, others simply create all flips themself, for days and weeks in a row.

Also, if idena prices eventually grows, the risk of partially or fully AI-produced flips will grow as well. Human farmers probably use every possible way to make flip production more efficient and less time demanding. 

And if we look at invitation rewards, it is obvious that human farms can extract value much easier again than individual solo-miners, if no real invitees are need to be chosen.

So currently 53% of session rewards as well as all mining rewards can theoretically exclusively be extracted by human farm owners only, while they have no necessary incentive to involve and invite real people or to let unique people produce flips.

I think that has turned out to be one major flaw in current idena cryptonomics.

so i am thinking about two IIPs here.

1. Reduce invitation rewards a bit further, maybe halve it.

It may harm growth, but human farmers extracting most invitation rewards with selfinvites harm real growth as well?

2. Make rewards for produced flips time critical. Reward only flips that where submitted within one hour of the beginning of each new epoch.

The idea is to force human farms to involve more farm workers into flip production, which will at last double the production costs from the perspective of farm owner but double the rewards from the perspektive of farm workerd and teach them also more about the benefits of becoming solo miners. 

Downsides: Many people, who just take a short brake in daily routines to do the idena session, wont have the time to produce flips right away but will loose those flip rewards as well. Also Farms are less likely to be profitable and therefore do less 'free' marketing for idena with their work. Also they could try to work only with preproduced reportworthy flips with one keyword. But i think the report system is getting better and better and could tackle with that, so that might be a less profitable decicion than to involve more if not all farm workers into flip production.

A nice byproduct of time critical flip production would be, that all those people running two or three identities will also have a much harder time to do that profitably, because one hour to produce 6 to 9 flips is much shorter than one hour to produce 3 flips.

Also flips will be much more semi-unique, much more creative and less AI-attackable, if we force human farms to involve more people in Flipproduction.

What else speaks against IIPs here in your opinion?

**Comment by Zen-44** [comment]
I believe making flips time critical will push others (individual identities) into rushing the production to receive flip rewards. This will lead to some using templates for their flips like the wake up routine or the born -> death cycle because they can't come with some other sotry or simply just want to get it done in that moment to at least receive some coins for it.

**Reply by ubiubi18** [reply]
I did check a few solo miner identities. many of them produce 3 flips in a row in less than one hour, but take often more than 10 minutes before they submit the next flip. Human farms behave often different, they submit faster, in a hurry!

In epoch 89 human farmers did take usually about 3 to 5 minutes to (come up with(?) &) submit flips for hours in a row. Each qualified flip gave a reward of 9,5 idna.

20 flips an hour give you 9,5idna x 20 = 190 , which even at this low marketcap equals an hourly wage of 6,43 $ ! IIP5 will not change that. There is a high incentive for human farmers to produce those flips faster to get done with it and earn more each hour, which might result for example in the use of those "wake up" templates.

The risk of solo-miners or family pools creating flips in more rush is neglectable in comparsion to the status quo where human farms have high incentive to create flips in a hurry by design. Solo-miners often take 10 minutes or more before they submit the next flip, but they still can do it easily within one hour - or a few hours.

So i still think flip production should become time-critical. Another way would be shrinking the rewards for flip-production in general, but that would be another blow to late adopters without high idna stake. So it seems more convenient to me to keep or even double the flip rewards (in favor of solo-miner late adopters), but shrink the available time (to mitigate the risk of centralized value extraction)!

**Reply by ubiubi18** [reply]
checking manually like 10 random delegated addresses from our "wake up"-template  indonesian pool ( https://scan.idena.io/address/0x96d11da40FDe82D81ebE0EAE61bFe6a47F43d1a6 ) surprise: 2 minutes each flip or even less So it could potentially submit (produce?) 30 flips each hour => 285 idna => 9,66$ / hour 
this extraction route wont stop with iip5, as long as enough farm workers are willing to work for even lower salary
